clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -2270,10 +2276,14 @@ public Metrics metricsRegistry() {
     }
 
     @Override
-    public AsyncConsumerMetrics kafkaConsumerMetrics() {
+    public KafkaConsumerMetrics kafkaConsumerMetrics() {
         return kafkaConsumerMetrics;
     }
 
+    public AsyncConsumerMetrics asyncConsumerMetrics() {
```

Comment:
I don't think this needs to be public. The test is in the same package. For `kafkaConsumerMetrics()`, there is a test which accesses the metrics object from the external object via the delegate.